3.0.0/reference.json

```diff
@@ -138,13 +138,27 @@
             "default-value": 0,
             "type":"float",
             "default-meaning": "The layer will be visible at the minimum possible scale",
-            "doc": "The minimum scale denominator that this layer will be visible at. A layer's visibility is determined by whether its status is true and if the Map scale >= minzoom - 1e-6 and scale < maxzoom + 1e-6."
+            "doc": "The minimum scale denominator that this layer will be visible at. A layer's visibility is determined by whether its status is true and if the Map scale >= minzoom - 1e-6 and scale < maxzoom + 1e-6. This option has been deprecated in favor of `minimum-scale-denominator`.",
+            "status": "deprecated"
```

Comment:
Was it deprecated in `3.0.0` yet?

Reply:
Take a look on https://github.com/mapnik/mapnik/commit/219ad1f2ccd2eaa5799dad73236378b127f490d9, there are all git tags including all 3.0.x versions.

Reply:
Thanks :)